feature-group-definitions/text-fragments.yml

```diff
@@ -0,0 +1,7 @@
+name: Text fragments
+description: Text fragments allow linking directly to a snippet of text within a page, causing it to be scrolled into view and/or highlighted. The linked text is specified in a URL fragment on the form `#:~:text=snippet`.
```

Comment:
I'm not excited about the "allows" or "and/or" here. I've tried breaking up the thoughts more—what do you think?

```suggestion
description: Text fragments link to a snippet of text within a page. The browser may scroll, highlight, or otherwise bring that text to the reader's attention. The linked text is specified in a URL fragment on the form `#:~:text=snippet`.
```

Reply:
I like that, I'll apply the suggestion and then tweak "The linked text is specified" further.